### PR TITLE
Added 1.19-7 docs

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by TerryHowe — APPROVED** [review]
/lgtm

File: docs/contents/index.md [hunk 1]
```diff
@@ -128,6 +128,7 @@ if you are interested in the AL2 container runtime kernel version.
 * [v1-20-eks-1](releases/1-20/1/index.md) (May 18, 2021)
 
 #### EKS-D 1.19 Version Dependencies
+* [v1-19-eks-7](releases/1-19/7/index.md) (July 26, 2021)
```
Comment (kschumy) [inline docs/contents/index.md:131]:
```suggestion
* [v1-19-eks-7](releases/1-19/7/index.md) (July 27, 2021)
```

**Pull request merged**